### PR TITLE
Typo and old sentences

### PR DESCRIPTION
**Pull request by gork7777**
On Android you can not download, you can install on it and it should be Install.
"Attach technical logs" should be "Send detailed system info" because Android and Windows using this, not sure other platforms.

### CODE REVIEW

File: docs/adguard-vpn-for-mac/solving-problems/logs.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ By default, AdGuard VPN for Mac uses the standard logging level, that is, the ba
 
 3. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred. If you can't reproduce the problem, specify as accurately as possible when it last occurred.
 
-4. There is a check mark next to the **Attach technical logs**, which means that when you send a report, you also send logs.
+4. There is a check mark next to the **Send detailed system info**, which means that when you send a report, you also send logs.
```
Comment (anastasiia-fedotova) [inline docs/adguard-vpn-for-mac/solving-problems/logs.md:18]: image

As you can see the current wording is correct

File: docs/adguard-vpn-for-mac/solving-problems/logs.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ By default, AdGuard VPN for Mac uses the standard logging level, that is, the ba
...
 
->If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, select **Settings** → **General** → **Export logs** in the **Actions** section on the right.
+>If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, go to **Support** → **Export logs and system info**.
```
Comment (anastasiia-fedotova) [inline docs/adguard-vpn-for-mac/solving-problems/logs.md:20]: image
As you can see the current wording is correct

File: docs/adguard-vpn-for-windows/solving-problems/logs.md [hunk 3]
```diff
@@ -39,6 +39,6 @@ In most cases, the default logging level is sufficient to trace down possible bu
 
 6. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred.
 
-7. Make sure that there is a check mark next to **Attach technical logs** and click **Send**.
+7. Make sure that there is a check mark next to **Send detailed system info** and click **Send**.
 
->If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, select **Settings** → **General** → **Export logs** in the **Actions** section on the right.
\ No newline at end of file
+>If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, go to **Support** → **Export logs and system info**.
```
Comment (anastasiia-fedotova) [inline docs/adguard-vpn-for-windows/solving-problems/logs.md:44]: I've checked - all the wording is consistent with what we use in our apps. If you want us to change the wording, write us via Crowdin.

**Review by anastasiia-fedotova — CHANGES_REQUESTED** [review]

File: docs/adguard-vpn-for-windows/solving-problems/logs.md (hunk 3 above)
Comment (gork7777) [reply docs/adguard-vpn-for-windows/solving-problems/logs.md:44]: @anastasiia-fedotova I already reported in Crowdin as a mistake issue you can look at these and also sent a message.
As you can see its correct for Windows: 

image

File: docs/adguard-vpn-for-mac/solving-problems/logs.md (hunk 1 above)
Comment (anastasiia-fedotova) [reply docs/adguard-vpn-for-mac/solving-problems/logs.md]: @gork7777 I wrote down your suggestions and will pass them on to our content team. But in any case, wordings should be changed in the apps first, and then in the Knowledge Base.

**Review by anastasiia-fedotova — CHANGES_REQUESTED** [review]

File: docs/adguard-vpn-for-mac/solving-problems/logs.md [hunk 4]
```diff
@@ -35,8 +35,8 @@ In most cases, the default logging level is sufficient to trace down possible bu
 
 6. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred.
 
-7. Make sure that there is a check mark next to **Attach technical logs** and click **Send**.
+7. Make sure that there is a check mark next to **Send detailed system info** and click **Send**.
 
->If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, select **Settings** → **General** → **Export logs** in the **Actions** section on the right.
+>If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, go to **Support** → **Export logs and system info**.
```
[the remark this answers is not in the document]
Comment (gork7777) [reply docs/adguard-vpn-for-mac/solving-problems/logs.md]: @anastasiia-fedotova I reverted the changes in this commit and check again: https://github.com/AdguardTeam/KnowledgeBaseVPN/pull/49/commits/349e4751cf1f23364f17fc6b9825fcd0846c397a

**Pull request merged**